### PR TITLE
de (German) translation

### PR DESCRIPTION
**Pull request by LeSnake04**
Commit made via [Stringlate](https://lonamiwebs.github.io/stringlate/)

### CODE REVIEW

**Comment by LeSnake04** [comment]
100% translated

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="app_name">2050</string>
+    <string name="app_version">v.%1$s</string>
+    <string name="app_copyright">© 2019 Matthew Chandler</string>
+    <string name="app_license">Dieses Programm hat absolut keine gewärleistung.\n
```
Comment (reinerh) [inline app/src/main/res/values-de/strings.xml]: Gewährleistung

File: app/src/main/res/values-de/strings.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <string-array name="theme_settings" tools:ignore="InconsistentArrays">
+    </string-array>
+    <string name="loc_perm_msg">\"Du wirst nach Standortzugriff gefragt werden. \n
+ \n
+ Wenn du den Zugriff erlaubst, wird dein Standort genutzt, um den Tag / Nachtmodus zu wechseln. Ansonsten wird die Uhr deines Gerätes verwendet.\n
+ \n
+ Dein Standort wird nicht gespeichert oder andersartig genutzt.</string>
+
+
+    <string name="help_general">Kombiniere die Bälle mit der Gleichen Nummer und Farbe wie, um den 2048 ball zu erreichen, während du verhinderst, dass die Bälle sich zu weit voneinander entfernen.</string>
```
Comment (reinerh) [inline app/src/main/res/values-de/strings.xml]: gleichen
Ball

File: app/src/main/res/values-de/strings.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <string name="help_touch">Wische mit deinem Finger, um die Bewegungsrichtung der Bälle zu ändern. Bei jeder Richtungsänderung wird ein neuer ball generiert.</string>
+    <string name="help_accel">Drehe dein Gerät so, dass die Bälle runterfallen. Neue Bälle werden generiert, sobald du das gerät drehst.</string>
```
Comment (reinerh) [inline app/src/main/res/values-de/strings.xml]: Gerät

File: app/src/main/res/values-de/strings.xml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    </string-array>
+    <string name="loc_perm_msg">\"Du wirst nach Standortzugriff gefragt werden. \n
+ \n
+ Wenn du den Zugriff erlaubst, wird dein Standort genutzt, um den Tag / Nachtmodus zu wechseln. Ansonsten wird die Uhr deines Gerätes verwendet.\n
+ \n
+ Dein Standort wird nicht gespeichert oder andersartig genutzt.</string>
+
+
+    <string name="help_general">Kombiniere die Bälle mit der Gleichen Nummer und Farbe wie, um den 2048 ball zu erreichen, während du verhinderst, dass die Bälle sich zu weit voneinander entfernen.</string>
+    <string name="help_touch">Wische mit deinem Finger, um die Bewegungsrichtung der Bälle zu ändern. Bei jeder Richtungsänderung wird ein neuer ball generiert.</string>
```
Comment (reinerh) [inline app/src/main/res/values-de/strings.xml:90]: Ball

File: app/src/main/res/values-de/strings.xml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="app_name">2050</string>
+    <string name="app_version">v.%1$s</string>
+    <string name="app_copyright">© 2019 Matthew Chandler</string>
+    <string name="app_license">Dieses Programm hat absolut keine gewärleistung.\n
+  Sehen sie die <a href="https://opensource.org/licenses/mit-license.php" >MIT License (MIT)</a> für Details an.</string>
```
Comment (reinerh) [inline app/src/main/res/values-de/strings.xml]: In den anderen Texten verwendest du "du", hier aber "sie".

**Comment by mattvchandler** [comment]
Sorry for taking so long to see this. I haven't looked here in a while and github didn't send any notification.
I appreciate the contribution!
There are a few issues that should be resolved before merging, though.

- [x] res/values/values.xml should not be translated. I thought I had marked all of those as translatable="false", but apparently not (I'll fix that soon). These strings are keys, not user-facing, so the theme menu won't work correctly if they have the incorrect values.
- [x] res/values-v21/strings.xml needs to be translated.
- [x] There are a few values in res/values/strings.xml that weren't translated:
  * theme_settings
  * loc_perm_title
  * achieve_texts - These are the toast messages that pop up when you first reach a larger size ball. They don't need to be translated literally, so feel free to have some fun. The first 3 shouldn't ever be shown to the user, but are there in case something breaks. The 12th is shown when 2048 is reached.
- [x] Some of the translations are split across lines. I don't think it affects the output, but I'd like to see them on one line
  * app_license
  * loc_per_msg
- [x] Please squash your commits into a single commit

I will gladly merge this once the above are fixed. Thank you!

**Comment by mattvchandler** [comment]
Continued in PR #12

**Pull request closed without merging**